warehouse/rss/views.py

```diff
@@ -72,3 +73,34 @@ def rss_packages(request):
     )
 
     return {"newest_projects": newest_projects}
+
+
+@view_config(
+    route_name="rss.user_updates",
+    context=User,
+    renderer="rss/user_updates.xml",
+    decorator=[
+        origin_cache(
+            1 * 24 * 60 * 60,  # 1 day
+            stale_while_revalidate=1 * 24 * 60 * 60,  # 1 day
+            stale_if_error=5 * 24 * 60 * 60,  # 5 days
+            keys=["all-projects"],
```

Comment:
This line would cause the cache for every user RSS feed to be purged whenever any new project was created. It's not necessary since we already have the `context=User` line above.

Reply:
Thanks for the catch.  I just copied and pasted this from the other RSS views, should have read through the code to understand it first.